src/test/compile-fail/issue-36082.rs

```diff
@@ -8,18 +8,23 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+// revisions: ast mir
+//[mir]compile-flags: -Z emit-end-regions -Z borrowck-mir
+
 use std::cell::RefCell;
 
 fn main() {
     let mut r = 0;
     let s = 0;
     let x = RefCell::new((&mut r,s));
 
-    let val: &_ = x.borrow().0;
-    //~^ ERROR borrowed value does not live long enough
+    let val: &_ = x.borrow().0; //[ast]~ ERROR borrowed value does not live long enough [E0597]
+    //[mir]~^ ERROR borrowed value does not live long enough (Ast) [E0597]
+    //[mir]~^ ERROR borrowed value does not live long enough (Mir) [E0597]
```

Comment:
This needs to be `//[mir]~|`, I believe.

Reply:
Resolved this, my bad.